eng/helix/helix.proj

```diff
@@ -19,7 +19,7 @@
   <PropertyGroup Condition=" '$(CI)' == 'true' ">
     <HelixType>ci</HelixType>
     <Creator>aspnetcore</Creator>
-    <HelixBuild>$(BUILD_BUILDNUMBER)</HelixBuild>
+    <HelixBuild>$(BUILD_BUILDNUMBER)-$([System.DateTime]::Now.ToString('HH:mm'))</HelixBuild>
```

Comment:
```suggestion
    <HelixBuild>$(BUILD_BUILDNUMBER).$([System.DateTime]::Now.ToString('HH.mm'))</HelixBuild>
```


Let's workaround a bug in the Helix SDK causing it to generate an incorrect url in the logs.


![image](https://user-images.githubusercontent.com/2696087/58273980-b1761780-7d46-11e9-8704-cb8b64ea17da.png)
https://dev.azure.com/dnceng/public/_build/results?buildId=197549&view=logs&j=1074ac01-8c55-5109-1085-a16f28133b94&t=c1387476-520f-5f83-69e0-e1e27b3446ff&l=1411

Helix says: https://mc.dot.net/#/user/aspnetcore/pr~2Faspnet~2Faspnetcore/ci/20190522.28.07~3A01
Actual URL: https://mc.dot.net/#/user/aspnetcore/pr~2Faspnet~2Faspnetcore/ci/20190522.28.07:01